demos/python_demos/facies_classification_demo/README.md

```diff
@@ -0,0 +1,46 @@
+# Facies classification Python* Demo
+
+This demo demonstrate how to run facies classification using OpenVINO&trade;
+
+
+## How It Works
+Upon the start-up, the demo application loads a network and an given dataset file to the Inference Engine plugin. When inference is done, the application displays 3d itkwidget viewer with facies interpretation.
+
+## Running
+
+### Installation of dependencies
+
+Steps to create CPU extension:
+
+```bash
+source /opt/intel/openvino/bin/setupvars.sh
```

Comment:
OV supports not root installation if I right remember (in home directory)
```suggestion
source <openvino_install>/bin/setupvars.sh
```
BTW, possibly it makes sense also provide instruction for windows